python/lsst/ip/isr/isrTask.py

```diff
@@ -585,6 +594,11 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
                 self.maskAndInterpNan(ccdExposure)
                 interpolationDone = True
 
+            if self.config.brighterFatterLevel == 'DETECTOR':
+                bfKernel = bfKernel[ccdExposure.getDetector().getId()]
+            else:
+                # TODO: DM-15631 for implementing this
+                raise NotImplementedError("per-amplifier brighter-fatter correction not yet implemented")
```

Comment:
Do you anticipate being able to call `self.brighterFatterCorrection` in a single pass for the per-amplifier case? If not, I suggest putting the following into the "if" above.